components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodClientProjectSessionTracker.kt

```diff
@@ -32,12 +33,11 @@ import java.util.concurrent.CancellationException
 import java.util.concurrent.CompletableFuture
 
 @Suppress("UnstableApiUsage", "OPT_IN_USAGE")
-class GitpodClientProjectSessionTracker(
-        private val session: ClientProjectSession
-) : Disposable {
+class GitpodClientProjectSessionTracker(private val project: Project) : Disposable {
```

Comment:
Reminder: As we changed the signature: `private val session: ClientProjectSession` -> `private val project: Project` we made it compatible with 223.7x but incompatible with 222.x.
We could create wrappers for these classes (one in `latest` and one in `stable`) package, which calls the class from the root passing the `Project` object.

Reply:
Done ✅